cmd/collectors/restperf/restperf.go

```diff
@@ -812,6 +1161,14 @@ func (r *RestPerf) PollInstance() (*matrix.Matrix, error) {
 	return nil, err
 }
 
+func isWorkloadObject(query string) bool {
```

Comment:
do you use these two guard clauses seperately? if not, maybe combine into `isQos()` and use that everywhere?

Reply:
yes used at seperate places hence 2 methods.